zips/zip-0312.rst

```diff
@@ -228,6 +191,14 @@ that the given SIGHASH matches the data sent from the Coordinator, or compute th
 SIGHASH themselves from that data. However, the specific mechanism for that process
 is outside the scope of this ZIP.
 
+In Zcash, the `randomizer` is called :math:`\mathsf{alpha}` and MUST be
+generated accordingly to the Zcash specification [#protocol-spendauthsig]_. Note
+that the choice of :math:`\mathsf{alpha}` influences the SIGHASH computation, so
+it is impossible to compute the `randomizer` based on the message (SIGHASH), as
+suggested in [#frost-rerandomized]_. This is not an issue as long the
+`randomizer` is generated uniformly at random, which is done in the Zcash
+specification.
```

Comment:
It's an overspecification to imply that the randomizer needs to be generated uniformly at random. As suggested above, it could be chosen depending on the commitments sent in round 1.

Also, the spec actually says to generate the randomizer like this:

> Define $\mathsf{RedDSA.GenRandom} ⦂ () \rightarrow \mathsf{RedDSA.Random}$ as:
> * Choose a byte sequence $T$ uniformly at random on $\mathbb{B}^{\mathbb{Y}[(\ell_{\mathsf{H}} + 128) / 8]}$.
> * Return $\mathsf{H}^⊛(T)$.

To refine my suggestion, $T$ would be the "seed" and then the randomizer would be computed by all participants as $\mathsf{H}^⊛(T \hspace{0.2em}\mathsf{||}\hspace{0.2em} \mathsf{commitments}).$

Reply:
Great idea. I went ahead with it in 62833e6c